fe/fe-core/src/main/java/com/starrocks/http/HttpServerHandler.java

```diff
@@ -150,7 +151,7 @@ private void writeResponse(ChannelHandlerContext context, HttpResponseStatus sta
     }
 
     @Override
-    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
```

Comment:
A tiny improvement here, no need to be part of this PR

add debug log to print the exception stack. Several times we saw http server excepted, but didn't know where is the exception point, it is nice to be able to turn on debug log here to log the stack trace for easier debug and fix.

Reply:
https://github.com/StarRocks/starrocks/issues/40316 @nshangyiming

Reply:
copy that